src/main/java/com/minecolonies/coremod/colony/CitizenData.java

```diff
@@ -234,6 +244,7 @@ public void readFromNBT(@NotNull final NBTTagCompound compound)
         name = compound.getString(TAG_NAME);
         female = compound.getBoolean(TAG_FEMALE);
         paused = compound.getBoolean(TAG_PAUSED);
+        isChild = compound.getBoolean(TAG_CHILD);
```

Comment:
Please surround this new getBoolean with a contains check.

Reply:
Actually it returns false automatically when it can't find it afaik

Reply:
Ah okey, that might then be because of auto unboxing.
If you do this with a compound it throws an NPE.